juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
+    topic external {
```

Comment:
Please put appropriate topic name as per the domain

Reply:
Done